### PR TITLE
[4.3] ca operator: allow for a non-existent router ca configmap

### PR DESCRIPTION
**Pull request by csrwng**
4.3 backport of #83

### CODE REVIEW

**Comment by KodieGlosserIBM** [comment]
Same question as here: https://github.com/openshift/hypershift-toolkit/pull/83#issuecomment-579391477

File: assets/ca-operator/ca-operator-deployment.yaml [hunk 1]
```diff
@@ -67,17 +67,22 @@ spec:
...
-               continue
-            fi
             if ! oc get cm -n openshift-config-managed service-ca -o jsonpath='{ .data.ca-bundle\.crt }' > /tmp/service.ca; then
                echo "Cannot fetch service-ca yet. Will continue to wait"
                sleep 30
                continue
             fi
-            cat /etc/kubernetes/config/initial-ca.crt /tmp/router.ca /tmp/service.ca > /tmp/kcm.ca
+            include_router_ca=1
+            if ! oc get cm -n openshift-config-managed router-ca -o jsonpath='{ .data.ca-bundle\.crt }' > /tmp/router.ca; then
```
Comment (relyt0925) [inline assets/ca-operator/ca-operator-deployment.yaml:76]: if this errors due to a network error what is the failure case?

**Pull request merged**